src/applications/personalization/profile/tests/e2e/contact-information/home-address-modal.cypress.spec.js

```diff
@@ -79,53 +76,4 @@ describe('Home address update modal', () => {
 
     cy.injectAxeThenAxeCheck();
   });
-
-  it('should show update prompt modal and show error when updating fails', () => {
```

Comment:
Why did we delete this test?

Reply:
@adamwhitlock1 moved this test into a new file - `contact-information/home-address-modal-failure.cypress.spec.js`